### PR TITLE
Added wrapper for Highs_setSolution()

### PR DESCRIPTION
**Pull request by yongqli**
The doc comments could probably be improved, please feel free to edit it.

### CODE REVIEW

**Comment by mmghannam** [comment]
Hi @yongqli, thanks for your contribution! Could you please add tests? For example you can set a solution with value 50 to a model and solving with zero time limit and making sure its primal bound is 50.

File: src/lib.rs [hunk 1]
```diff
@@ -451,6 +452,73 @@ impl Model {
...
+    }
+
+    /// Tries to hot-start using an initial guess by passing the column and row primal and dual solution values.
+    /// See highs_c_api.h for further details.
+    ///
+    /// # Panics
+    ///
+    /// If the data passed in do not have the correct lengths.
+    /// `cols` and `col_duals` should have the lengths of `num_cols`.
+    /// `rows` and `row_duals` should have the lengths of `num_rows`.
```
Comment (lovasoa) [inline src/lib.rs:483]: Since this returns an error type, wouldn't it be better to return a clean error instead of panicking ?

**Comment by KnorpelSenf** [comment]
> I don't think so. If you want to start where @yongqli left off, that would be very welcome !

I'll see what I can do :)

File: src/lib.rs (hunk 1 above)
Comment (KnorpelSenf) [reply src/lib.rs:483]: This is just a documentation error, isn't it?

**Comment by KnorpelSenf** [comment]
@lovasoa I opened #22.
